src/table/table.scss

```diff
@@ -122,265 +251,316 @@
         }
       }
 
-      &.iui-tables-head {
-        @include themed {
-          background-color: t(iui-color-background-3);
-        }
-        user-select: none;
+      // #region Indented rows
+      &.iui-expander-indent-0 {
+        padding-left: 0;
+      }
 
-        .iui-icon {
-          @include themed {
-            fill: t(iui-icons-color-actionable);
-          }
+      &.iui-expander-indent-1 {
+        padding-left: $iui-xl;
+      }
 
-          &:hover {
-            @include themed {
-              fill: t(iui-icons-color-actionable-hover);
-            }
-          }
-        }
+      &.iui-expander-indent-2 {
+        padding-left: $iui-xl * 2;
+      }
 
-        > .iui-filter {
-          margin-left: $iui-sm;
-          cursor: pointer;
+      &.iui-indent-1 {
+        padding-left: $iui-xl + $iui-s;
+      }
 
-          > .iui-icon {
-            width: $iui-icons-default;
-            height: $iui-icons-default;
-          }
-        }
+      &.iui-indent-2 {
+        padding-left: ($iui-xl + $iui-s) * 2;
+      }
+      // #endregion Indented rows
+    }
 
-        > .iui-sort {
-          flex-grow: 1;
-          justify-content: flex-end !important;
+    // #region Selection
+    &:not(.iui-selected) + .iui-selected {
+      padding-bottom: $iui-xxs;
+      border-bottom: none;
+    }
 
-          > .iui-icon-wrapper {
-            cursor: pointer;
+    &.iui-selected {
+      padding: 0;
+      border-bottom: none;
+      @include themed {
+        border: solid 1px t(iui-color-foreground-primary);
```

Comment:
This rule overrides the border-bottom: none.. What is the intention here?

Reply:
I still do not really get...border-bottom: none is overriden with the next rule, so my point is, it sohuld not be needed in there...

Reply:
It's all about limitation on how selector work, making rule that won't move the content or change the size of the row. 
Line 287 : `border: solid 1px t(iui-color-foreground-primary);` Set full border
Line 285 : `border-bottom: none;` remove bottom-border
Line 300-308 : `border-top: solid 1px rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-4));` set border-top on the following row.

![image](https://user-images.githubusercontent.com/4070713/120670421-66775a80-c45e-11eb-8a3c-8dbceda9c490.png)
I can't help you more than that.

Reply:
So my point was, that border-bottom: none is not needed in this case (does nothing), cause its overriden with the next row rule.
![image](https://user-images.githubusercontent.com/81580355/120676322-91819e80-c49e-11eb-8fe9-6f37691c6373.png)

ALso, all border rules can be simplified by changing just border-color versus overiding whole border (when you actually change only color).

Reply:
You still don't understand... Let me clear. I don't need border-bottom, but there is 2 way to make the same thing : 
```scss
border: solid 1px t(iui-color-foreground-primary);
border-bottom: none;
```
vs.
```scss
border-top: solid 1px t(iui-color-foreground-primary);
border-left: solid 1px t(iui-color-foreground-primary);
border-right: solid 1px t(iui-color-foreground-primary);
```
I prefer nullify one rule with 2 rules than defining 3 rules

Reply:
Please pay attention to my screenshot above and the "No need" text. Just check up plain css in here and tell me, what it gives:
![image](https://user-images.githubusercontent.com/81580355/120680482-08209b00-c4a3-11eb-95a2-214020d84c12.png)

Unless I really do not know something in css. I would appreciate if you could create a simple sandbox, that these exact order rules could have different influence, than just having a single rule (without border-bottom: none).

Reply:
I'm sorry, I think you might be on something here. Seem like this rule is unneeded because line 265 and line 286 are covering what line 270 cover and cannot cover. 

I guess, we can remove this rule at line 270 as there is better rule for doing the same :
&.iui-selected { ~~border-bottom: none;~~ }